sycl/test-e2e/Matrix/slm_utils.hpp

```diff
@@ -0,0 +1,81 @@
+//==------------------ slm_utils.hpp  - DPC++ joint_matrix------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+template <unsigned int colsA, unsigned int colsB, unsigned int MCache2,
+          unsigned int NCache2, unsigned int KCache2, unsigned int vnniFactor,
+          size_t SGs, typename TOperand, access::address_space Space>
```

Comment:
Some parameter names start from lowercase, some from uppercase. Could you please fix?

Reply:
Some of them are acronyms like SG, or specify the shape M, N, K, I prefer to keep them upper case.

Reply:
cols and Space are not acronyms but use different casing. vnni and SG are acronyms but use different casing. Can it be aligned?